presto-kudu/src/main/java/io/prestosql/plugin/kudu/KuduClientConfig.java

```diff
@@ -151,4 +155,18 @@ public boolean isGroupedExecutionEnabled()
     {
         return groupedExecutionEnabled;
     }
+
+    @NotNull
+    public Duration getDynamicFilteringProbeBlockingTimeout()
+    {
+        return dynamicFilteringProbeBlockingTimeout;
+    }
+
+    @Config("kudu.dynamic-filtering-probe-blocking-timeout")
```

Comment:
Probe is pretty technical term. Also, in theory, DF can be a result of something else.

what about
```
kudu.dynamic-filtering.wait-timeout
```

?

Reply:
Modified